recipe/meta.yaml

```diff
@@ -20,7 +20,8 @@ build:
 
 requirements:
   build:
-    - python {{ environ['PY_VER'] + '*' }}  # [win]
+    # Add python for feature activation
+    - python  # [win]
```

Comment:
@dschreij you need to rerender after add this. Let me try a PR to your feedstock to demonstrate it.